sygnal.yaml.sample

```diff
@@ -169,3 +169,10 @@ apps:
   #  # the default is 20.
   #  #max_connections: 20
 
+db:
```

Comment:
this should probably be nearer the top? and needs some comments. What does it do? what are the defaults if the settings are not given?